### PR TITLE
feat: add web wrapper for running as a standalone app

### PR DESCRIPTION
**Pull request by gilach**

### CODE REVIEW

File: packages/autocannon-ui-desktop/.eslintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+cli.js
```
Comment (simoneb) [inline packages/autocannon-ui-desktop/.eslintignore:1]: why no eslint?

File: packages/autocannon-ui-web/cli.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env node
+
+// eslint-disable-next-line strict
```
Comment (simoneb) [inline packages/autocannon-ui-web/cli.js:3]: can we use "use strict" instead of ignoring the rule?

**Review by simoneb — COMMENTED** [review]
what's up with the empty .npmignore file? I'm not sure it's doing what we want to do. shall we not instead use package.json fields to specify what we want to end up in the package?

I'm not seeing where the frontend is built. I guess the dist needs to be in the npm package, meaning that it should be built before it's published. the prepare script is often used for that purpose

**Comment by gilach** [comment]
> what's up with the empty .npmignore file? I'm not sure it's doing what we want to do. shall we not instead use package.json fields to specify what we want to end up in the package?

Since `.gitignore` file excludes `dist` folder, adding an empty `.npmignore` file will cause npm to not use `.gitignore` as a ruleset for exclusions, therefore `dist` folder will be included in the package. The other option, as you mentioned, would be to provide `files` clause in `package.json` marking `dist` and `index.js` to be included in the package. Any specific reason why we should go with `files` approach?

> I'm not seeing where the frontend is built. I guess the dist needs to be in the npm package, meaning that it should be built before it's published. the prepare script is often used for that purpose

Added.

**Comment by simoneb** [comment]
> > what's up with the empty .npmignore file? I'm not sure it's doing what we want to do. shall we not instead use package.json fields to specify what we want to end up in the package?
> 
> Since `.gitignore` file excludes `dist` folder, adding an empty `.npmignore` file will cause npm to not use `.gitignore` as a ruleset for exclusions, therefore `dist` folder will be included in the package. The other option, as you mentioned, would be to provide `files` clause in `package.json` marking `dist` and `index.js` to be included in the package. Any specific reason why we should go with `files` approach?
> 

Not a strong reason but I would say it is to make sure we include only what we need. The published package only needs the dist folder and index.js file, hence let's just specify those to avoid getting more stuff than we need in there.

> > I'm not seeing where the frontend is built. I guess the dist needs to be in the npm package, meaning that it should be built before it's published. the prepare script is often used for that purpose
> 
> Added.

File: package.json [hunk 3]
```diff
@@ -20,7 +20,7 @@
     "version:safe": "lerna version --no-git-tag-version --no-push",
     "pre-commit": "lerna run --concurrency 1 --stream precommit --since HEAD --exclude-dependents",
     "toc": "markdown-toc README.md -i",
-    "prepare": "husky install"
+    "prepare": "husky install && lerna run build --stream"
```
Comment (simoneb) [inline package.json:23]: put it in the frontend package.json, not here

**Review by simoneb — COMMENTED** [review]
Do we need to update the docs or nothing needed for this change?

**Comment by gilach** [comment]
> Do we need to update the docs or nothing needed for this change?

README is already updated. Nothing else needed I guess.

**Review by simoneb — APPROVED** [review]

**Review by simoneb — APPROVED** [review]

**Pull request merged**